seed/static/seed/partials/insights_property.html

```diff
@@ -0,0 +1,159 @@
+<div class="page_header_container" ng-cloak>
+    <div class="page_header">
+        <div class="left page_action_container"></div>
+        <div class="page_title"><h1 translate>Property Insights</h1></div>
+        <div class="right page_action_container"></div>
+    </div>
+</div>
+
+<div class="section_nav_container">
+    <div class="section_nav" ng-include="::urls.static_url + 'seed/partials/insights_nav.html'"></div>
+</div>
+
+
+<div class="section_content_container">
+  <div class="section_content container-fluid">
+    <!-- <div class="content_block row" style="padding:20px 20px 0px 20px">
```

Comment:
Can this be removed?

Reply:
I think this is a placeholder for the export functionality. No harm in keeping it.